eds_scikit/__init__.py

```diff
@@ -118,14 +119,18 @@ def improve_performances(
 
     to_add_conf.extend(
         [
-            ("spark.app.name", f"{os.environ.get('USER')}_scikit"),
+            ("spark.app.name", f"{os.environ.get('USER')}_{app_name}_scikit"),
             ("spark.sql.session.timeZone", tz),
             ("spark.sql.execution.arrow.enabled", "true"),
             ("spark.sql.execution.arrow.pyspark.enabled", "true"),
         ]
     )
 
     for key, value in to_add_conf:
+        if not conf.contains(key):
```

Comment:
I'm not sure if we should do this, by default some parameters will not be present in the conf even though they correspond to actual parameters

Reply:
You're right. I think it would be useful to check parameters correctness though. I remove it.